modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -222,6 +336,28 @@ bool HttpSynchronization::trySyncFromUrl(std::string listUrl) {
     }
     startedAllDownloads = true;
 
+    constexpr int MaxDownloadRetries = 5;
+    int downloadTry = 0;
+    bool downloadFailed = false;
```

Comment:
I'd probably go with `downloadSucceeded` instead, since that is the normal behavior.
```suggestion
    bool downloadSucceeded = false;
```